src/components/Loading.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Loading = () => {
```

Comment:
do you see this animation loading?

Reply:
it is just a plus..

Reply:
No I dont see animation loading yet. Probably its my stretch task which I will do later.